app/controllers/appeals_controller.rb

```diff
@@ -92,6 +92,18 @@ def document_count
     handle_non_critical_error("document_count", error)
   end
 
+  # series_id is lowercase, no curly braces because it comes from url
+  def document_lookup
+    series_id = "{#{params[:series_id]}}".upcase
+    document = Document.find_by(series_id: series_id, file_number: appeal.veteran_file_number)
+
+    if !document
+      document = VBMSService.fetch_document_series_for(appeal).map(&:series_id).include?(series_id)
+    end
+
+    render json: { document_presence: !!document }
```

Comment:
```suggestion
    render json: { document_presence: document.present? }
```

This suggestion and the previous one are there just to reduce the number of negation operators. The code as-is works fine, but I think it could be good to simplify the logic a bit.

Reply:
Love both of those changes, I committed both of them. I knew the `if !document` could've been better. The usage of unless always slips my mind though